src/borg/archiver.py

```diff
@@ -1307,8 +1309,11 @@ def do_debug_dump_manifest(self, args, repository, manifest, key):
 
         meta = self.debug_json(msgpack.fallback.unpackb(data, object_hook=StableDict, unicode_errors='surrogateescape'))
 
-        with open(args.path, 'w') as fd:
-            json.dump(meta, fd, indent=4)
+        if args.path == '-':
+            json.dump(meta, sys.stdout, indent=4)
+        else:
+            with open(args.path, 'w') as fd:
```

Comment:
Yes, that would be useful in many places. But i wonder if that might also contains decent error reporting etc. That would be quite nice for many simple commands.